changes/31.8.1.md

```diff
@@ -6,3 +6,17 @@
 * Optimize glyph for Cyrillic Lower Dzze (`U+A689`) under italics.
 * Optimize glyphs for Volapük Ae/Oe/Ue (`U+A79A`..`U+A79F`).
 * Optimize glyph for Latin Lower Dezh Digraph with Palatal Hook (`U+1DF12`).
+* Add characters:
```

Comment:
Since now we added new featurs, please bump the version to 31.9.0 as we are better enforcing semver. Rename this file to `31.9.0.md`.